datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/UpgradeCliApplication.java

```diff
@@ -34,7 +36,9 @@
             KafkaEventConsumerFactory.class,
             GraphQLEngineFactory.class,
             KafkaTraceReaderFactory.class,
-            TraceServiceFactory.class
+            TraceServiceFactory.class,
+            KafkaConsumerPoolFactory.class,
```

Comment:
Need to exclude the factories here due to the requirement to load all factories....

Reply:
include?

Reply:
The datahub-upgrade image is used for a lot of different things. The no-code process requires a lot of the factories and so it imports everything. There are however things that it definitely doesn't need or require and so we're adding them here. Hopefully at some point we might deprecate that no-code process, not actually sure what it does these days.